source/Serial/NetworkSerial.cpp

```diff
@@ -155,20 +155,20 @@ NetworkSerial::flush(
     create_task( _current_store_operation )
```

Comment:
I still don't understand why _current_store_operation is a member variable. Why not use a lambda capture? In this case, you have some curious behavior if the user issues multiple flush() commands quickly.